packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -54,4 +57,21 @@ export class MeterProvider implements api.MeterProvider {
 
     return this._meters.get(key)!;
   }
+
+  shutdown(cb: () => void = () => {}) {
+    this.onShutdown().then(() => {
+      setTimeout(cb, 0);
+    });
+  }
+
+  private async onShutdown() {
```

Comment:
onShutdown sounds like an event handler, but you are using it directly above on 62. A better name might be `shutdownAllMeters` or `shutdownChildren`.

Reply:
There was a slight issue with the suggestion you had for the meter shutdown, I fixed that and the naming of the functions.